### ISSUE TITLE
no-outline-none false positive

### ISSUE THREAD

**Issue by fluky** [YozhikM/stylelint-a11y#44]
```
tag:focus:not(:focus-visible) {
   outline: none;
}
```

This triggers the no-outline-none rule.  This syntax specifically targets browsers that support focus-visible and have determined that the user is a mouse user.
